install.sh

```diff
@@ -15,30 +22,39 @@ src/gz routing http://archive.openwrt.org/chaos_calmer/15.05.1/brcm63xx/smp/pack
 src/gz packages http://archive.openwrt.org/chaos_calmer/15.05.1/brcm63xx/smp/packages/packages
 src/gz telephony http://archive.openwrt.org/chaos_calmer/15.05.1/brcm63xx/smp/packages/telephony
 EOF
-echo "Added appropriate lists into distfeeds.conf"
+logger "Done"
 
+logger "Updating opkg..."
 opkg update
-echo "Updating opkg"
+logger "Done"
 
+logger "Downloading recompiled packages..."
 curl -Lk https://github.com/nutterpc/tg-luci/tarball/master --output /tmp/tg-luci.tar.gz
-echo "Downloaded the .ipk files"
+logger "Done"
+
+logger "Untarring..."
 mkdir /tmp/tg-luci
 tar -xzf /tmp/tg-luci.tar.gz -C /tmp/tg-luci
-echo "Untarred the .ipk files"
+logger "Done"
+
+logger "Installing LuCI depends..."
 rm /tmp/tg-luci.tar.gz
 opkg --force-reinstall --force-overwrite --force-checksum install /tmp/tg-luci/*/*.ipk
-echo "Installing LuCI dependancies using opkg"
 rm -rf /tmp/tg-luci
+logger "Done"
 
+logger "Installing LuCI..." 
 opkg install luci luci-lib-json luci-lib-jsonc luci-lib-nixio luci-mod-rpc
-echo "Installed LuCI"
+logger "Done"
 
```

Comment:
plese add here `cp /rom/usr/lib/lua/uci.so /usr/lib/lua/` to fix https://github.com/nutterpc/tg-luci/issues/13

Reply:
if you want to add it Francyesco, go for it. I'm unavailable today (it's my birthday)

Otherwise I'll be able to start on the issues tomorrow :)

Reply:
Happy bday :P
btw you can ignore this edit... a different approach is needed explained here https://github.com/nutterpc/tg-luci/issues/13#issuecomment-426074194